Modules/Loadable/VolumeRendering/Logic/vtkSlicerVolumeRenderingLogic.cxx

```diff
@@ -292,48 +292,48 @@ void vtkSlicerVolumeRenderingLogic::ChangeVolumeRenderingMethod(const char* disp
 void vtkSlicerVolumeRenderingLogic::UpdateVolumeRenderingDisplayNode(vtkMRMLVolumeRenderingDisplayNode* node)
 {
   if (!node)
-  {
+    {
```

Comment:
we should avoid reverting back to the old style

Reply:
I've fixed it now (I still had the old brace style in my original working copy).